test/fuzzing/consensus_fuzz.cpp

```diff
@@ -48,18 +45,17 @@ namespace fuzzing {
         : timer_(std::make_shared<iroha::consensus::yac::MockTimer>()),
           cleanup_strategy_(std::make_shared<
                             iroha::consensus::yac::BufferedCleanupStrategy>()),
-          client_creator_([](const shared_model::interface::Peer &peer) {
-            return std::make_unique<
-                iroha::consensus::yac::proto::MockYacStub>();
-          }),
           keypair_(shared_model::crypto::DefaultCryptoAlgorithmType::
                        generateKeypair()),
           async_call_(std::make_shared<
                       iroha::network::AsyncGrpcClient<google::protobuf::Empty>>(
               logger::getDummyLoggerPtr())),
           initial_round_{1, 1} {
       network_ = std::make_shared<iroha::consensus::yac::NetworkImpl>(
-          async_call_, client_creator_, logger::getDummyLoggerPtr());
+          async_call_,
+          std::make_unique<iroha::network::MockClientFactory<
```

Comment:
Why no expectations are set for this client factory? It should return nullptrs by default, which does not sound right. Can you try running fuzzing tests?